server/game/cards/02_SHD/units/OmegaPartOfTheSquad.ts

```diff
@@ -43,12 +43,15 @@ export default class OmegaPartOfTheSquad extends NonLeaderUnitCard {
     }
 
     private isFirstClonePlayedByControllerThisPhase(card) {
-        const clonesPlayedByThisPlayerThisPhase = this.cardsPlayedThisPhaseWatcher.getCardsPlayed((playedCardEntry) =>
-            playedCardEntry.playedBy === card.controller &&
-            playedCardEntry.card.hasSomeTrait(Trait.Clone) &&
-            playedCardEntry.card !== card
-        );
-        return clonesPlayedByThisPlayerThisPhase.length === 0;
+        if (card.hasSomeTrait(Trait.Clone)) {
```

Comment:
Could we streamline this a bit and reduce nesting by checking the if condition first:

```typescript
if (!card.hasSomeTrait(Trait.Clone)) {
    return false;
}

// check watcher below
```

Reply:
shuffled it